ui/src/plugins/i18n.js

```diff
@@ -0,0 +1,31 @@
+import Vue from 'vue';
+import VueI18n from 'vue-i18n';
+import en from './en.json';
+import es from './es.json';
+import hi from './hi.json';
+import kr from './kr.json';
+import ru from './ru.json';
+import sc from './zh_Hans_CN.json';
+import zh from './zh_Hans_TW.json';
+import ar from './ar';
+import vn from './vn';
+
+Vue.use(VueI18n);
+
+const messages = {
+  en: en,
+  es: es,
+  hi: hi,
+  kr: kr,
+  ru: ru,
+  sc: sc,
+  zh: zh,
+  ar: ar,
+  vn: vn
+};
+
+export default new VueI18n({
+  locale: 'en',
```

Comment:
I think you should set the default locale based on the browser language. 

https://github.com/kazupon/vue-i18n/issues/220

I have my firefox browser (not my ina browser) enabled for Spanish, English, & German to test internationalization. You should try that.